server/datastore/inmemory/inmemory.go

```diff
@@ -0,0 +1,14 @@
+package inmemory
```

Comment:
Since we have our database implemented, we don't need this file anymore. You can delete it.